### PR TITLE
feat(pricelists): schema for new pricelists endpoint

### PR DESCRIPTION
**Pull request by tatiana-sudakova**

### CODE REVIEW

**Review by mohsen89z — CHANGES_REQUESTED** [review]
LGTM, some minor changes.
Also, it's missing the changes in [products index](https://github.dev/tillhub/schemas/blob/65d8bd3f2b91ed8735a3ed1758db965403e1a72c/lib/v1/products/index.js)

File: lib/v1/products/pricelist/response.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    currency: {
+      ...oneOf({
+        type: 'string',
+        description: 'The three letter [ISO currency](https://en.wikipedia.org/wiki/ISO_4217) of this item.',
+        example: 'EUR',
+        minLength: 3,
+        maxLength: 3
+      })
+    },
+    price_book: oneOf({
```
Comment (alghanor) [inline lib/v1/products/pricelist/response.js:69]: @tatiana-sudakova should not be in client response

File: lib/v1/products/pricelist/response.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        'time-based'
+      ],
+      example: 'scaled'
+    },
+    created_at: {
+      type: 'string',
+      format: 'date-time',
+      example: '2018-11-04T23:18:43.075Z'
+    },
+    updated_at: {
```
Comment (alghanor) [inline lib/v1/products/pricelist/response.js:27]: @tatiana-sudakova should not be in client response

File: lib/v1/products/pricelist/response.js (hunk 1 above)
Comment (tatiana-sudakova) [reply lib/v1/products/pricelist/response.js:69]: That was in tech spec, I can remove it.

File: lib/v1/products/pricelist/response.js (hunk 2 above)
Comment (tatiana-sudakova) [reply lib/v1/products/pricelist/response.js:27]: @alghanor 
Why are you against updated_at field? I think it's necessary for client to check that data they received are newer then the one they have in local cache.

**Review by mohsen89z — APPROVED** [review]

**Pull request merged**